pycqed/instrument_drivers/physical_instruments/QuTech_CCL.py

```diff
@@ -78,22 +74,14 @@ def _initialize_insn_microcode_parsers(self):
         """
         The parser helper objects are initialized in this function.
         """
-        self.microcode = CCLightMicrocode()
+        self.CCL_microcode = CCLightMicrocode()
         self.QISA = QISA_Driver()
         self.QISA.enableScannerTracing(False)
         self.QISA.enableParserTracing(False)
         self.QISA.setVerbose(False)
 
-    def stop(self):
-        self.run(0),
-        self.enable(0)
-
-    def start(self):
-        self.enable(1)
-        self.run(1),
-
     def add_parameter(self, name,
-                      parameter_class=Parameter, **kwargs):
+                      parameter_class=StandardParameter, **kwargs):
```

Comment:
Can you undo this replacement? It has to do with a recent update to QCoDeS, StandardParameter is now depracated. If you pull the latest version of QCoDeS from DiCarloLab-Delft/QCoDes/master you should have the right version.